src/utils/dom.js

```diff
@@ -78,3 +79,26 @@ export function focus(root: ?HTMLElement, selector?: string, focusRoot: boolean
         root.focus();
     }
 }
+
+/**
+ * Scrolls the container / modal / wrapper instead of the body
+ *
+ * @param {HTMLElement} itemEl - the base dom element to search
+ * @return {void}
+ */
+export function scrollIntoView(itemEl: ?HTMLElement): void {
+    // @NOTE: breaks encapsulation but alternative is unknown child ref
+    if (itemEl) {
+        const parentEl = itemEl.closest('html, .modal, .overlay-wrapper');
+        scrollIntoViewIfNeeded(itemEl, false, undefined, parentEl);
+    }
+}
+
+window.Element.prototype.closest = function closest(s) {
```

Comment:
Move this to the top of the file and put a nice comment on top of it, and guard it as shown here https://developer.mozilla.org/en-US/docs/Web/API/Element/closest#Polyfill

I also think you may need the matches part as shown in the link above due to https://caniuse.com/#search=matches IE11 using ms prefix